docs/csharp/language-reference/compiler-options/how-to-set-environment-variables-for-the-visual-studio-command-line.md

```diff
@@ -35,7 +35,14 @@ If the current version of Visual Studio is installed on a computer that also has
   
     > [!CAUTION]
     >  VsDevCmd.bat can vary from computer to computer. Do not replace a missing or damaged VsDevCmd.bat file with a VsDevCmd.bat from another computer. Instead, rerun setup to replace the missing file.  
-  
+
+### Available options for VsDevCmd.BAT
+
+To see the available options for VsDevCmd.BAT, run the command with the `-help` flag:
```

Comment:
May replace `flag` with parameter instead.

Reply:
I'd prefer "option". I made the quick fix.